src/i18n/strings/en_EN.json

```diff
@@ -1796,5 +1796,7 @@
     "If you didn't remove the recovery method, an attacker may be trying to access your account. Change your account password and set a new recovery method immediately in Settings.": "If you didn't remove the recovery method, an attacker may be trying to access your account. Change your account password and set a new recovery method immediately in Settings.",
     "Failed to set direct chat tag": "Failed to set direct chat tag",
     "Failed to remove tag %(tagName)s from room": "Failed to remove tag %(tagName)s from room",
-    "Failed to add tag %(tagName)s to room": "Failed to add tag %(tagName)s to room"
+    "Failed to add tag %(tagName)s to room": "Failed to add tag %(tagName)s to room",
+    "Read Marker lifetime (ms)": "Read Marker lifetime (ms)",
+    "Read Marker off-screen lifetime (ms)": "Read Marker off-screen lifetime (ms)"
```

Comment:
we stopped adding to the end for a number of reasons: run `yarn i18n` to fix the build error.